### ISSUE TITLE
|mispgetevent is giving errors, does not work

### ISSUE THREAD

**Issue by ag-michael** [remg427/misp42splunk#159]
`mispgetevent` is not working, for reference a command like this should work per the documentation:

```
|mispgetevent misp_instance=testing_mispinput type="domain,hostname"
```
but I am getting this error 
```
External search command 'mispgetevent' returned error code 1. Script output = "error_message=AttributeError at "/opt/splunk/etc/apps/misp42splunk/bin/packages/aob_py2/splunklib/searchcommands/reporting_command.py", line 89 : 'function' object has no attribute 'ConfigurationSettings' ".
```

**Comment by remg427** [comment]
Hi,
which version do you use? mispgetevent should now be a Generating command. Also one param from json_request, last, eventid, date is required

**Comment by ag-michael** [comment]
@remg427  It turns out, it does work but we get that ugly error message anyways.  Can we leave this issue open and get that error resolved? I'm using version 3.1.12

**Comment by remg427** [comment]
fixed in 3.2.2

**Comment by remg427** [comment]
resolve in >= 3.2.2
